src/utils/call-binding.ts

```diff
@@ -0,0 +1,35 @@
+import { AppExpandLevels } from '../constant/apps';
+import { MattermostClient } from '../clients';
+import { AppCallRequest, Channel, MattermostOptions } from '../types';
+import manifest from '../manifest.json';
+
+export const callBindingByApp = async (call: AppCallRequest, path: string) => {
```

Comment:
This is necessary due to, after the user has linked their Google Drive account, in the same method is requested to call another method to start the notifications

Reply:
@liz-segura98 As stated above, this may not be supported in the future. It may need to be explicitly supported through the OAuth scope effort currently in progress. cc @hanzei @levb